modules/custom-status/custom-status.php

```diff
@@ -662,6 +690,11 @@ public function get_custom_statuses() {
 					$status->$key = $value;
 				}
 			}
+
+			// Add term meta
+			$required_user_ids         = get_term_meta( $status->term_id, 'required_user_ids', true );
+			$status->required_user_ids = '' === $required_user_ids ? [] : $required_user_ids;
```

Comment:
The term's `required_user_ids` property is filled in automatically here from term meta.